components/locale.js

```diff
@@ -61,14 +61,26 @@ function getInitialLocaleProps({req} = {}) {
   return Promise.resolve({locale})
 }
 
-// eslint-disable-next-line complexity
 function getContent(locale, options) {
+  const fallbackContent = content('', options)
+  if (locale === fallbackLocale) {
+    return fallbackContent
+  }
+  const localeContent = content(`${locale}/`, options)
+  return Object.keys(fallbackContent).reduce((cont, key) => {
```

Comment:
I'm thinking that we'll want to do a deep merge here to handle Arrays ([like here](https://github.com/kentcdodds/glamorous-website/blob/83a8d767a80dbd80724e573014bcc5c614949956/pages/basics/content/index.js#L6-L13)).